packages/styles/scss/components/content-item-horizontal-media/_content-item-horizontal-media.scss

```diff
@@ -0,0 +1,162 @@
+/**
+ * Copyright IBM Corp. 2016, 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+@import '../../globals/imports';
+@import '../link-list/index';
+@import 'carbon-components/scss/globals/scss/vendor/@carbon/elements/scss/layout/breakpoint';
+
+@mixin content-item-horizontal-media {
+  :host(#{$dds-prefix}-content-item-horizontal-media),
+  .#{$prefix}--content-item-horizontal__item {
+    @include carbon--make-row;
+
+    border-top: 1px solid $ui-03;
+
+    @include carbon--breakpoint(sm) {
+      padding-top: $carbon--spacing-07;
+      padding-bottom: $carbon--spacing-07;
+    }
+
+    @include carbon--breakpoint(md) {
+      min-height: rem(320px);
+      padding-top: $carbon--spacing-07;
+      padding-bottom: $carbon--spacing-09;
+    }
+
+    @include carbon--breakpoint(lg) {
+      min-height: rem(296px);
+      padding-top: $carbon--spacing-07;
+      padding-bottom: $carbon--layout-05;
+    }
+
+    padding-left: $carbon--spacing-05;
+    padding-right: $carbon--spacing-05;
+  }
```

Comment:
Just to double-check - Did you go back to `carbon--make-row` because of the gutter thing, or something else? If it's the gutter thing, we can use [`gap`](https://developer.mozilla.org/en-US/docs/Web/CSS/gap).